.cirrus.yml

```diff
@@ -391,6 +391,111 @@ task:
       type: text/plain
 
 
+linux_fedora_rawhide_template: &linux_fedora_rawhide_template
+  env:
+    CPUS: 4
+    BUILD_JOBS: 4
+    TEST_JOBS: 8 # experimentally derived to be a decent choice
+
+    CCACHE_DIR: /tmp/ccache_dir
+    DEBUGINFOD_URLS: "https://debuginfod.fedoraproject.org/"
+
+  only_if: $CIRRUS_CHANGE_MESSAGE !=~ '.*\nci-os-only:.*' || $CIRRUS_CHANGE_MESSAGE =~ '.*\nci-os-only:[^\n]*linux.*'
```

Comment:
Could you mark it as a trigger_type: manual? For many changes we don't need to retest all OSs...